src/library/scala/collection/MapView.scala

```diff
@@ -27,6 +27,7 @@ trait MapView[K, +V]
    *
    *  @return the keys of this map as a view.
    */
+  @nowarn
```

Comment:
Why add a `@nowarn` after putting it as deprecated ?

Reply:
It's deprecated for overriding, but we can't remove it from subclasses, in particular where it can leak to client code, e.g., `mymapview.keys`.